sql/cagg_utils.sql

```diff
@@ -0,0 +1,31 @@
+-- This file and its contents are licensed under the Apache License 2.0.
```

Comment:
There is a possibility that it should be licensed under TLS. Please correct me if I'm wrong.

Reply:
That's a good question. My guess is that all `.sql` files were Apache, and the C code would either implement the functionality or not.

Reply:
I added this question to our internal "Database Team Development Process" document.